.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -171,6 +171,7 @@ jobs:
           CIRCLE_SHA1: ${{ github.event.pull_request.head.sha || github.sha }}
           CIRCLE_TAG: ${{ steps.parse-ref.outputs.tag }}
           CIRCLE_WORKFLOW_ID: '${{ github.run_id }}_${{ github.run_number }}'
+          GITHUB_WORKFLOW_RUN_ID: '${{ github.run_id }}'
```

Comment:
Isn't this already a thing?

`GITHUB_RUN_ID` should give you the same info and it's already built in 

https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables